modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -20,17 +20,17 @@ To create a backup, run the following command on any node in the cluster:
 gravity backup /var/lib/gravity/planet/share/backup.tar.gz
 ----
 
-This command creates an archive of the current system state in: `/var/lib/gravity/planet/share/backup.tar.gz`. You can pass the following optional parameters to the `gravity` backup command:
+This command creates an archive of the current system state in: `/var/lib/gravity/planet/share/backup.tar.gz`. You can pass the following optional parameters to the gravity backup command:
 
 * `--follow`
 +
-Outputs the backup job logs to stdout.
```

Comment:
This is still valid statement. `follow` outputs the backup logs to stdout. Maybe worth to mention that `tee` in the example above is a way to output in stdout and file at the same time.

Reply:
@UnderGreen thanks. Updated again